test/test/console-log.tap

```diff
@@ -18,9 +18,10 @@ TAP version 13
         ok 4 - nested ok second
         1..4
     ok 2 - second ___/# time=[0-9.]+(ms)?/~~~
-ok 1 - nesting ___/# time=[0-9.]+(ms)?/~~~
 
 >>>> after second child
+ok 1 - nesting ___/# time=[0-9.]+(ms)?/~~~
+
```

Comment:
They are. `>>>> after second child` is logged from inside the `nesting` test. It cannot be ended after `second` completes because it may still return a promise. Note that ` >>>> after child test` correctly follows `ok 1 - nesting`.

[Source](https://github.com/tapjs/node-tap/blob/2543379a8bb7eeea2161263241ada4c58c47b54d/test/test/console-log.js#L32)